e2e/nomostest/ntopts/multi_repo.go

```diff
@@ -55,6 +56,9 @@ type MultiRepo struct {
 
 	// RepoSyncPermissions will grant a list of PolicyRules to NS reconcilers
 	RepoSyncPermissions []rbacv1.PolicyRule
+
+	// DeletionPropagationPolicy sets the deletion propagation policy for all RSyncs
+	DeletionPropagationPolicy metadata.DeletionPropagationPolicy
```

Comment:
Do we want these to be a pointer so you can set it to nil to omit the annotation?

Reply:
done